src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs

```diff
@@ -350,7 +349,12 @@ public RenameDriver CreateFieldRenamer(RecordType parameters, DPath pathToRename
             ** but that we don't return any display names for them. Thus, we clone a PowerFxConfig but without 
             ** display name support and construct a resolver from that instead, which we use for the rewrite binding.
             */
-            return new RenameDriver(parameters, pathToRename, updatedName, this, CreateResolverInternal(), CreateBinderGlue(), culture);
+            return new RenameDriver(parameters, pathToRename, updatedName, this, CreateResolverInternal() as ReadOnlySymbolTable, CreateBinderGlue(), culture, false);
```

Comment:
>as ReadOnlySymbolTable [](http://example.com/codeflow?start=106&length=22)

This could potentially break existing paths if CreateFieldRenamer does not return ROST. 

But 
1.  that can only happen in cases where Engine.CreateREsolver is derived - which is internal. Probably just a PA scenario, and PA doesn't call this method anyways. 
2. OptionSetRename would still be broken. 

So it's probably ok.